components/eamxx/cmake/machine-files/ruby.cmake

```diff
@@ -0,0 +1,13 @@
+include(${CMAKE_CURRENT_LIST_DIR}/common.cmake)
+common_setup()
+
+# Load all kokkos settings from Ekat's mach file
+set(Kokkos_ENABLE_SERIAL TRUE CACHE BOOL "")
+
+# Enable Broadwell arch in Kokkos
+option(Kokkos_ARCH_BDW "" ON)
```

Comment:
Unlike quartz, ruby is unknown to EKAT, so you will need to do a bit of extra setup here compared to the eamxx quartz.cmake file. The EKAT quartz file has:

```
# Load Broadwell flags and openmp backend for kokkos
include (${CMAKE_CURRENT_LIST_DIR}/kokkos/intel-bdw.cmake)
include (${CMAKE_CURRENT_LIST_DIR}/kokkos/openmp.cmake)

include (${CMAKE_CURRENT_LIST_DIR}/mpi/srun.cmake)
```

You'll probably want some of that here and you'll need to use EKAT_MACH_FILES_PATH instead of CMAKE_CURRENT_LIST_DIR to refer to EKAT cmake files.